recipes/capypdf/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+import os
+
+required_conan_version = ">=2.0"
+
+
+class PackageConan(ConanFile):
+    name = "capypdf"
+    description = "Fully color-managed PDF generation library"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jpakkane/capypdf"
+    topics = ("pdf", "rendering")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    implements = ["auto_shared_fpic"]
+
+    def configure(self):
+       self.options["libtiff/*"].jpeg = 'libjpeg-turbo'
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libpng/[>=1.6 <2]")
+        self.requires("lcms/2.16")
+        self.requires("freetype/2.13.2")
+        self.requires("libtiff/4.7.0")
+
+    def build_requirements(self):
+        self.build_requires("meson/1.7.2")
+        self.build_requires("pkgconf/2.2.0")
+
+    def validate(self):
+        check_min_cppstd(self, 23)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = MesonToolchain(self)
+        tc.generate()
+        deps = PkgConfigDeps(self)
+        deps.generate()
+
+    def build(self):
+        meson = Meson(self)
+        meson.configure()
+        meson.build()
+
+    def package(self):
+        copy(self, "LICENSE", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        meson = Meson(self)
+        meson.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
```

Comment:
```suggestion
        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
        folders = os.listdir(os.path.join(self.package_folder, "lib",))
        python_folder = [f for f in folders if f.startswith("python") and os.path.isdir(os.path.join(self.package_folder, "lib", f))]
        rmdir(self, os.path.join(self.package_folder, "lib", python_folder[0]))
```

Let's remove that folder that keeps a Python cache file.

Reply:
The thing is, we  can not track the Python API in the package, as the package ID will be represented by settings, covered mostly by C and C++. As the Python API may depend of the python version used, it may work or not for who is consuming a pre-built package. I would keep out any python cache/api from the final package.

Reply:
Thanks for the explanation. I already deleted the whole python directory anyway in the last push.